### PR TITLE
Add git commit to status() method & build

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

**Review by ialarmedalien — APPROVED** [review]

File: src/us/kbase/workspace/gitcommit/GitCommit.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	
+	// can't really test this easily since the file must be baked into the jar or war,
+	// just test manually
+
+	/** The Git commit from which the service was built. */
+	public static final String COMMIT;
+	
+	private static final String COMMIT_FILE_NAME = "gitcommit";
+	
+	static {
```
Comment (Xiangs18) [inline src/us/kbase/workspace/gitcommit/GitCommit.java:23]: how is static {} different from `static` keyword before var or method

Comment (MrCreosote) [reply src/us/kbase/workspace/gitcommit/GitCommit.java:23]: A static block runs when the class is imported for the first time. A static variable or method are class variables / methods

Comment (MrCreosote) [reply src/us/kbase/workspace/gitcommit/GitCommit.java:23]: https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html

Comment (Xiangs18) [reply src/us/kbase/workspace/gitcommit/GitCommit.java:23]: Make sense now!!

**Review by Xiangs18 — APPROVED** [review]
LGTM!

**Pull request merged**